Automattic/jetpack
# PHPCS: Carousel

A partial fix for #21960

The remaining fix is at this PR: https://github.com/Automattic/jetpack/pull/22835

#### Changes proposed in this Pull Request:

* This PR updates the carousel module to meet PHPCS standards:
   - "projects/plugins/jetpack/modules/carousel.php" 
   - "projects/plugins/jetpack/modules/carousel/jetpack-carousel.php" 

#### Jetpack product discussion

n/a

#### Does this pull request change what data or activity we track or use?

No

#### Testing instructions:

* Try testing changing carousel settings (Jetpack > Settings > Writing, and also at Settings > Media (/wp-admin/options-media.php) )
* Try testing adding a carousel to a test site
* Whilst doing the above, check for any broken experiences or PHPCS errors
